src/robopatrol/scheduler.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+import time
+import requests
+import json
+from apscheduler.schedulers.background import BackgroundScheduler
+
+poll_interval_sec = 5
+my_job_interval_sec = 3600
+my_job_name = 'my_job'
+configURL = 'http://192.168.115.1:9998/schedule'
+
+def my_job():
+    print('Tick! The time is: %s' % datetime.now())
```

Comment:
Here we should get the current job config and check that the job config was not deleted.